content/doc/developer/forms/form-validation.adoc

```diff
@@ -45,6 +45,21 @@ public FormValidation doCheckThreads(@QueryParameter("value") final String meani
 
 This naming convention is how Jenkins wires up your form validation method. The return type also must be FormValidation, which offers various static factory methods to create an instance. Form validation can be of 3 different kinds: `ok`, `warning` or `error`. Validation message can either be represented in plaintext or use HTML markup. Several validations of different kinds can be reported wrapping then into one using aggregate factory method.
 
+=== Setting up form validation within a Jelly template
+
+To configure form validation within a Jelly template to call `doCheckXxx` methods, defined in a particular class in Jenkins, you need to utilize the `checkUrl` and `checkDependsOn` attributes.
+Here's a breakdown of how these attributes are used:
+
+The `checkUrl` attribute specifies the URL that Jenkins requests to perform form validation.
+Typically, this URL points to the Jenkins server where it handles the validation request.
+The value entered in the input field is checked via AJAX against this URL and any errors encountered are rendered under the text field.
+This URL is associated with the method responsible for performing the validation (`doCheckXxx` method) within your Descriptor class.
+
+The `checkDependsOn` attribute defines the dependency between the field being validated and the field(s) whose values are required for validation.
```

Comment:
More accurately should be something like "In the modern `checkUrl` mode, which as of Jenkins 2.360 requires the `checkDependsOn` attribute to be set (but it can be an empty string). This mode automatically adds the value of the current form element as the query parameter called `value`. 

The proposed change "The `checkDependsOn` attribute defines the dependency between the field being validated and the field(s) whose values are required for validation." seems awkward and not to-the-point.